src/components/index.js

```diff
@@ -1,5 +1,6 @@
 import Header from "./Header";
 import Footer from "./Footer";
 import ProductCard from "./ProductCard";
+import Filter from "./Filter";
```

Comment:
you can set here export no need to write line number 6

Reply:
I will make the required changes.